drivers/bridge/bridge.go

```diff
@@ -357,6 +358,13 @@ func (d *driver) configure(option map[string]interface{}) error {
 		}
 		// Make sure on firewall reload, first thing being re-played is chains creation
 		iptables.OnReloaded(func() { logrus.Debugf("Recreating iptables chains on firewall reload"); setupIPChains(config) })
+
+		// Add DOCKER-USER chain
+		arrangeUserFilterRule()
```

Comment:
now arrangeUserFIlterRule is called on all systems, linux/windows/bsd etc, is this by intention?

Reply:
AFAIK `bridge` is for linux only